service/errors/errors.go

```diff
@@ -38,6 +38,7 @@ var (
 	ErrCodeSlackChannelNotSetForProject            = "ERR_SLACK_CHANNEL_NOT_SET_FOR_PROJECT"
 	ErrCodeGitTagNotFound                          = "ERR_GIT_TAG_NOT_FOUND"
 	ErrCodeGithubReleaseNotFound                   = "ERR_GITHUB_RELEASE_NOT_FOUND"
```

Comment:
`ErrCodeGithubReleaseNotFound` is not deleted because it will be used again in https://github.com/jan-zabloudil/release-manager/pull/120